components/footer.js

```diff
@@ -10,20 +10,44 @@ library.add(fab, faFacebookF, faTwitter);
 
 function Footer({ children }) {
     return (
-        <footer className={css.footer}>
-            <a href='https://www.facebook.com/AweAndReverence' title='Awe & Reverence on Facebook' target='_blank'>
-                <div className={css.facebook}>
-                    <FontAwesomeIcon icon={['fab', 'facebook-f']} />
-                </div>
-            </a>
+        <>
+            <link
```

Comment:
While we're using FontAwesome here in the footer, the `<link>` tag is supposed to belong in the `<head>` section of an HTML document, semantically.

You're going to want to move this tag to the HTML the correct file in the head section, which I believe is here: https://github.com/aweandreverence/www.aweandreverence.com/blob/fcba8f5178e1ce7891a0e140fb06f55e165532ff/components/base_page.js#L25